app/src/main/java/com/cmput301w23t40/capturetheqr/OtherPlayerActivity.java

```diff
@@ -36,6 +39,9 @@ protected void onCreate(Bundle savedInstanceState) {
             @Override
             public void onCallBack(QRCode maxQR, QRCode minQR) {
                 if(maxQR != null) {
+                    // FIXME check if the following two lines work after merging Anushka's PR
```

Comment:
I don't see a pr anymore for Anushka? Do these work? if so please remove the comment